astropy/cosmology/io/tests/test_ecsv.py

```diff
@@ -169,7 +169,7 @@ def test_fromformat_ecsv_subclass_partial_info(self, cosmo_cls, cosmo, read, wri
         # not equal, because Tcmb0 is changed
         assert got != cosmo
         assert got.Tcmb0 == cosmo_cls._init_signature.parameters["Tcmb0"].default
-        assert got.clone(name=cosmo.name, Tcmb0=cosmo.Tcmb0) == cosmo
+        assert got.clone(name=cosmo.name, Tcmb0=cosmo.Tcmb0, m_nu=cosmo.m_nu) == cosmo
```

Comment:
Why is this necessary? You didn't change `m_nu`. Maybe related to the override of `_init_arguments` above?

Note that this is not a big deal, but if there is a reason, add it to the comment on line 169.

(obviously, the same comment for all the other instances...)

Reply:
OK, I see now!